Applications/Games/Europa Universalis II/Local/script.js

```diff
@@ -0,0 +1,10 @@
+include(["Engines", "Wine", "QuickScript", "LocalInstallerScript"]);
+
+new LocalInstallerScript()
+    .name("Europa Universalis II")                       
+    .editor("Ubisoft")                               
+    .author("ImperatorS79") 
+    .category("Games")                           
+    .executable("eu2.exe")
+    //TO DO : find static link to 1.07 and 1.09 update, and add a .postinstall. 
```

Comment:
I would like to provide those updates, but can find a link that stay the same every time, if you see what I try to say...